libs/community/extended_testing_deps.txt

```diff
@@ -14,6 +14,7 @@ chardet>=5.1.0,<6
 cloudpathlib>=0.18,<0.19
 cloudpickle>=2.0.0
 cohere>=4,<6
+crate==1.0.0.dev1
```

Comment:
### About
Last summer, we removed the SQLAlchemy dialect from the fundamental HTTP driver package. The next release of the [crate](https://github.com/crate/crate-python) package, version 1.0.0, will conclude the transition. Afterwards, it will be fine to just pull in the new [sqlalchemy-cratedb](https://github.com/crate/sqlalchemy-cratedb) package.

### References
- https://github.com/crate/crate-python/pull/616
- https://github.com/crate/crate-python/issues/620

Reply:
We have been doing some yak shaving towards a 1.0.0 release.

- https://github.com/crate-workbench/langchain/issues/29

Reply:
Version 1.0.0 of crate-python has been released, e6ccd807bb updates this branch accordingly.